frontend/app/components/shared/LiveSessionSearchField/LiveSessionSearchField.tsx

```diff
@@ -30,7 +30,7 @@ function LiveSessionSearchField(props: Props) {
     <div className="relative">
       <Input
         // inputProps={ { "data-openreplay-label": "Search", "autocomplete": "off" } }
-        className={stl.searchField}
+        className={stl.searchField, {'h-8'}}
```

Comment:
unrelated

Reply:
plus we do not do that. add on top:

`import cn from 'classnames'`

then combine classes via 

`cn(stl.searchField, 'h-8')`

Reply:
Reverted this change. I'll work on a sep branch to address assist related changes.